docroot/modules/custom/va_gov_vba_facility/src/EventSubscriber/VbaFacilitySubscriber.php

```diff
@@ -0,0 +1,126 @@
+<?php
+
+namespace Drupal\va_gov_vba_facility\EventSubscriber;
+
+use Drupal\Component\Render\FormattableMarkup;
+use Drupal\core_event_dispatcher\EntityHookEvents;
+use Drupal\Core\Entity\EntityTypeManager;
```

Comment:
```suggestion
use Drupal\Core\Entity\EntityTypeManagerInterface;
```
If there is an interface available, that is the better choicce to use.